### PR TITLE
Adding accessibility a11 y for components

### PR DESCRIPTION
**Pull request by RashelleKosib**
Added accessibility to a variety of ui-core components. 
Fixed violations by adding aria-labels. 
Passing default aria-labels via meta files.
Added a script to the test command (dev mode).

### CODE REVIEW

**Comment by JoMarton** [comment]
@RashelleKosib 
This PR currently has no affect on the library since no change was made to the build process

File: packages/wix-ui-core/src/components/TimePicker/TimePicker.tsx [hunk 1]
```diff
@@ -51,6 +51,9 @@ export type TimePickerProps = Pick<InputProps, 'disabled' | 'readOnly'> & {
 
   /** custom width of component. Goes into inline style so any css distance value allowed */
   style?: React.CSSProperties;
+
+  /** aria-label - Accessibility */
+  'aria-label'?: string;
```
Comment (JoMarton) [inline packages/wix-ui-core/src/components/TimePicker/TimePicker.tsx]: `TimePicker` already supports `aria-label` through `InputProps`

File: packages/wix-ui-core/src/components/AddressInput/AddressInput.meta.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ addressInputMeta
       apiKey: '',
       lang: 'en',
       Client,
-      onSelect: noop
+      onSelect: noop,
+      'aria-label': 'adressinput'
```
Comment (Zombiefruit) [inline packages/wix-ui-core/src/components/AddressInput/AddressInput.meta.ts]: Not sure if relevant after Jonathan's comments, but "address" is missing a "d" here

Comment (Zombiefruit) [inline packages/wix-ui-core/src/components/AddressInput/AddressInput.meta.ts]: Not sure if relevant after Jonathan's comments, but "address" is missing a "d" here

**Comment by shlomitc** [comment]
is `aria-label` and other `aria-*` are mandatory for components?
I think it should be optional `?`

**Comment by JoMarton** [comment]
@shlomitc 
Sure they should be optional. Is there a place here it is added as required?

**Comment by shlomitc** [comment]
https://github.com/wix/wix-ui/pull/700/files#diff-7f91bea5eb002f2aa38dbd4ed389a957R180

**Comment by shlomitc** [comment]
sorry, it's proptypes which is ok

**Comment by RashelleKosib** [comment]
@shlomitc  it's optional as a prop, but I did gave it a default value (through meta files only) mainly because of how we test it with A11Y, as a stand-alone component.

**Comment by JoMarton** [comment]
@shlomitc 
That's the propTypes syntax. to make it required it must be `string.isRequired`.
The typescript props are `"aria-label"?: string`

**Pull request merged**